tests/update/patch_before_migration.pm

```diff
@@ -73,6 +79,53 @@ sub patching_sle {
     set_var("SYSTEM_PATCHED", 1);
 }
 
+sub install_package {
+    my @pk_list = split(/,/, get_var('PACKAGES'));
+    for my $pk (@pk_list) {
+        zypper_call "in -t package $pk";
+    }
```

Comment:
According to tests/installation/select_patterns_and_packages.pm:
#    PACKAGES=quota-nfs,-samba,-grub2  packages starting with - will be removed

It would be good if keep the same behavior here.

Reply:
Good suggestion, I will imporve it.